lib/toggl.js

```diff
@@ -181,11 +202,25 @@ const updateTogglGroups = async () => {
   console.table(groups);
 };
 
+const updateTogglTags = async () => {
+  const tags = await getWorkspaceTags();
+
+  tags.forEach(tag => {
+    TogglTag.findOrCreate({
+      where: {
+        id: tag.id,
+      },
+      defaults: definedFieldsOnly(tag, TogglTag),
+    });
+  });
```

Comment:
does this should be a promise? the same kind of function `updateTogglGroups` is a promise

Reply:
Yeah, `findOrCreate` does return a promise, but I don't really need to do anything with the response when it resolves, and if it rejects we probably want it to stop anyway.